swupd/data.go

```diff
@@ -31,79 +32,32 @@ type dbgConfig struct {
 	src    string
 }
 
-type config struct {
-	stateDir  string
+type swupdData struct {
```

Comment:
This is a config, not data. Can you rename this appropriately?

Reply:
Yeah, i know. I just dont want to call if config. That is exactly what I`ve been trying to remove all along do avoid confusion. Actually, I want to get rid of this file and keep config/config.go only, but that requires some more refactoring.